viewer/packages/pointclouds/src/PointCloudNode.ts

```diff
@@ -97,12 +97,8 @@ export class PointCloudNode extends THREE.Group {
     this._potreeNode.pointShape = value;
   }
 
-  get stylableObjects(): RawStylableObject[] {
-    return this._potreeNode.stylableObjects;
-  }
-
-  setObjectStyle(objectId: number, appearance: PointCloudAppearance): void {
-    this.potreeNode.setObjectStyle(objectId, appearance);
+  setObjectStyle(styledCollection: StylablePointCloudObjectCollection): void {
```

Comment:
This is a confusing name - `set` should only be used when there only can be one, i.e. when calling set will override the effect of previous invocations. Use `assignObjectStyle` (which you have below, so I'm not sure why there's also a setter).